FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,112 @@
+for cmd_line_arg in cmd_line_args:
+    if cmd_line_arg.contains("run_map::"):
+        var arg_and_value := cmd_line_arg.split("::")
+        # This should split into an array of size 2, containing the parameter name and its corresponding value
+        if arg_and_value.size() != 2:
+            return
+
+        map_override = arg_and_value[1]
+
+        func_godot_map.local_map_file = map_override
+        func_godot_map.verify_and_build()</code></pre>
+    <blockquote><i>NOTE: You can extend this to have more commands like God mode or Debug mode as your game needs, and use them when launching in editor.<br></i></blockquote>
+    <h2>Setting Up Your Editor</h2>
+    <p>In every map editor you'll need to specify both an <b>executable</b> and <b>parameters</b> in your launch or build options. The executable can be either the <b>exported binary of your game</b> or the <b>Godot Editor binary</b> if launching from a Godot project. To run from your game from the Godot project, be sure to use the <code>--path</code> parameter, as defined in <a href="https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html" target="_blank"><b>Godot's Command Line Interface options</b></a>.</p>
+    <h3>TrenchBroom</h3>
+    <p><a href="https://trenchbroom.github.io/manual/latest/#launching_game_engines"><b>The TrenchBroom Reference Manual</b></a> covers launch option configurations quite thoroughly. If you chose to use the Godot Editor for your launch executable, make sure to start your parameters with the <code>--path</code> command. You'll then add the <code>--run_map::</code> build command we defined earlier, using the built-in TrenchBroom variable <code>${MAP_BASE_NAME}</code> to get the name of the currently opened map file.</a>.</p>
+    <p>An example of set of parameters to launch from the Godot Editor will look something like this: <code>--path "C:\GAME DEV\FuncGodot\test_project" -- run_map::${MAP_BASE_NAME}.map</code></p>
+    <p>
+        To run your map from TrenchBroom, navigate to <code>Run > Launch Engine</code>.
+        Go to <code>Configure engines...</code>, add a game engine profile, and provide a name and path to the executable for this. For this example, we use the path to the Godot editor executable.
+        Once you're happy with your engine configuration, click close. You will now see this as an option you can select. Select it.
+        <p align="center"><img src="../images/tips_editorbuild_trenchbroom1.png"><br></p>
+        <p>Now when you press Launch your game should launch using the parameters you defined!</p>
+    </p>
+
+    <h3>NetRadiant Custom</h3>
+
+    <p>NetRadiant Custom utilizes a <code>default_build_menu.xml</code> file, located in your game's gamepack folder for its build processes. This file is automatically generated by NetRadiant Custom after setting up your gamepack configuration, though it needs some additional setup in order to work with your project.</p>
```

Comment:
[we actually create this, oops]<https://github.com/func-godot/func_godot_plugin/blob/67e647eab4f34c4715e54f9a3251379acd16a525/addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd#L219>)